react/week2/my-app/src/components/Counter.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+
+export function Counter() {
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    var timer = setTimeout(() => {
+      setCount(prev => prev + 1);
+    }, 1000);
+    return () => {
+      console.log('clean up');
```

Comment:
Always remember to remove console log on your final push.

Reply:
Sure, thanks for the review.